### PR TITLE
feat: add config.repo in envd

### PR DESCRIPTION
**Pull request by VoVAllen**
Signed-off-by: Jinjing.Zhou <allenzhou@tensorchord.ai>

This will add repo information to the image label. Thus server can clone this into containers when needed.

### CODE REVIEW

File: pkg/lang/frontend/starlark/config/config.go [hunk 1]
```diff
@@ -212,3 +213,16 @@ func ruleFuncEntrypoint(thread *starlark.Thread, _ *starlark.Builtin,
 	ir.Entrypoint(argList)
 	return starlark.None, nil
 }
+
+func ruleFuncRepo(thread *starlark.Thread, _ *starlark.Builtin,
+	args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var url, description string
+
+	if err := starlark.UnpackArgs(ruleRepo, args, kwargs, "url?", &url, "description?", &description); err != nil {
```
Comment (kemingy) [inline pkg/lang/frontend/starlark/config/config.go:221]:
```suggestion
	if err := starlark.UnpackArgs(ruleRepo, args, kwargs, "url", &url, "description?", &description); err != nil {
```

`url` should be required.

**Comment by gaocegege** [comment]
We need to update envd-server too for the change.

**Comment by VoVAllen** [comment]
> We need to update envd-server too for the change.

Yeah, and they are independent. Let's merge this first

**Comment by gaocegege** [comment]
LGTM

**Review by kemingy — APPROVED** [review]
/lgtm

**Pull request merged**